### PR TITLE
Small make/docker fixes

### PR DESCRIPTION
**Pull request by fyrchik**
Preparation for the #95 .

### CODE REVIEW

File: runner.sh [hunk 1]
```diff
@@ -222,7 +231,7 @@ if [ -n "$NEOBENCH_VOTE" ]; then
   ARGS+=(--vote)
 fi
 
-if [ -z "$SINGLE" ]; then
+if [ "$NEOBENCH_VALIDATOR_COUNT" -eq 1 ]; then
   make prepare
 else
```
Comment (AnnaShaleva) [inline runner.sh]: Add one more `elif` for 4 validators?

File: runner.sh [hunk 2]
```diff
@@ -15,10 +15,12 @@ export NEOBENCH_LOGGER=${NEOBENCH_LOGGER:-none}
 export NEOBENCH_TYPE=${NEOBENCH_TYPE:-NEO}
 NEOBENCH_FROM_COUNT=${NEOBENCH_FROM_COUNT:-1}
 NEOBENCH_TO_COUNT=${NEOBENCH_TO_COUNT:-1}
+export NEOBENCH_VALIDATOR_COUNT=${NEOBENCH_VALIDATOR_COUNT:-4}
 
 show_help() {
 echo "Usage of benchmark runner:"
-echo "   -s, --single                     Use single consensus node."
+echo "   -u, --count                      Consensus node count."
```
Comment (AnnaShaleva) [inline runner.sh:23]: Why `u`?

File: .docker/rpc/docker-compose.bench.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+services:
+  base-bench:
+    image: registry.nspcc.ru/neo-bench/neo-bench:bench
+    container_name: bench
+    volumes:
+      - ./out:/out:rw
+      - ../build/dump.$NEOBENCH_TYPE.$NEOBENCH_FROM_COUNT.$NEOBENCH_TO_COUNT.txs:/dump.txs
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ../rpc/tokencontract/token.nef:/tokencontract/token.nef:ro
+      - ../rpc/tokencontract/token.manifest.json:/tokencontract/token.manifest.json:ro
```
Comment (AnnaShaleva) [inline .docker/rpc/docker-compose.bench.yml:12]: New line at the end of file should be added.

**Review by AnnaShaleva — COMMENTED** [review]
README should be updated wrt these changes.

File: runner.sh (hunk 2 above)
Comment (fyrchik) [reply runner.sh:23]: coUnt

File: .docker/rpc/docker-compose.bench.yml (hunk 3 above)
Comment (fyrchik) [reply .docker/rpc/docker-compose.bench.yml]: Fixed.

File: runner.sh (hunk 2 above)
Comment (roman-khimov) [reply runner.sh:23]: `--validator`? `--cn`? `--count` is too vague to me.

Comment (fyrchik) [reply runner.sh]: Yes! One could've thought `-v` must already be taken, but no.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**